addons/website_sale/views/templates.xml

```diff
@@ -1507,6 +1493,66 @@
         </t>
     </template>
 
+    <template id="cta_wrapper" name="CTA Wrapper">
+        <div id="o_wsale_cta_wrapper" t-att-class="'d-flex flex-wrap align-items-center gap-2 my-4' + (' w-100 w-lg-auto' if website.product_page_image_width == '50_pc' else ' w-100') + (' pt-3' if not website.product_page_image_width in ['none', '100_pc'] and not is_view_active('website_sale.cta_wrapper_boxed') else ' ')">
```

Comment:
Let's create a `<t t-set="is_50_pc" t-value="website.product_page_image_width == '50_pc'"/>`, you can define it in the product template if needed elsewhere or on top of `cta_wrapper` template otherwise. 

Maybe a `t-attf-class` might be a bit easier to read, it would look like this with the new variables:
```xml
        <div
            id="o_wsale_cta_wrapper"
            t-attf-class="d-flex flex-wrap align-items-center gap-2 my-4 #{' w-100 w-lg-auto' if is_50_pc else ' w-100'} #{' pt-3' if not is_fullwidth_content and not is_view_active('website_sale.cta_wrapper_boxed') else ' '}"
        >
```
Don't forget to indent the content of this template with the website_sale guidelines